integrations-and-supported-tools/transformers/scripts/Neptune_Transformers.py

```diff
@@ -0,0 +1,61 @@
+import neptune.new as neptune
+from datasets import load_dataset, load_metric
+from transformers import (
+    AutoModelForSequenceClassification,
+    AutoTokenizer,
+    Trainer,
+    TrainingArguments,
+)
+from transformers.integrations import NeptuneCallback
+
+run = neptune.init_run(
+    api_token=neptune.ANONYMOUS_API_TOKEN,
+    project="common/quickstarts",
```

Comment:
Same question here: Should we enable anonymous logging for the Transformers project and use that here?

Reply:
Nope. That project is used for our demos, and we don't want dirty runs there.
However, the [huggingface-integration](https://app.neptune.ai/o/common/org/huggingface-integration/experiments?split=tbl&dash=charts&viewId=standard-view) project would be a better place to log runs than `quickstarts`

Reply:
Agreed, @normandy7 did mention doing that. That reason I didn't do that was because I couldn't log with anonymous token. If that access can be provided easily then certainly we should point there.